game/core/src/me/lihq/game/screen/elements/OrthogonalTiledMapRendererWithSprite.java

```diff
@@ -51,6 +71,18 @@ public void render()
     {
         beginRender();
 
+        /*
+        Convert the sprites list into a list of AbstractPerson to be sorted
+         */
+        List<AbstractPerson> people = new ArrayList<AbstractPerson>();
+
+        for (Sprite sprite : sprites)
+        {
+            people.add((AbstractPerson) sprite);
```

Comment:
I tried that, had the same issue with having them as NPCs

It doesnt like rendering them... I'll give it a go though